homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, name, location_id, client):
         """Initialize the TotalConnect status."""
         self._name = name
         self._location_id = location_id
+        self._unique_id = f"{location_id}"
```

Comment:
```suggestion
        self._unique_id = str(location_id)
```

Does it match the unique id requirements?

https://developers.home-assistant.io/docs/entity_registry_index#unique-id-requirements

Reply:
It's not explicitly on the acceptable list, but I think it counts:  the location ID is a unique database key on the TotalConnect server.  Users can't change it.